share/templates.d/99-generic/aarch64.tmpl

```diff
@@ -29,6 +29,9 @@ mkdir ${KERNELDIR}
     ## normal aarch64
     installkernel images-${basearch} ${kernel.path} ${KERNELDIR}/vmlinuz
     installinitrd images-${basearch} ${kernel.initrd.path} ${KERNELDIR}/initrd.img
+    %if doupgrade:
```

Comment:
doupgrade needs to be added to the <%page section's list of variables. It also needs to be passed into the template parser. eg. https://github.com/weldr/lorax/blob/27e611629f2c654939b0894086512e9d4f366113/src/pylorax/treebuilder.py#L258

Reply:
Err, oops. I didn't see it in your commit so I assumed it also needed to be restored. Apparently it's already there so nevermind :)